public/docs/_examples/hierarchical-dependency-injection/dart/lib/hero_editor_component.dart

```diff
@@ -0,0 +1,47 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'restore_service.dart';
```

Comment:
Ditto about using `package:` imports. I'll stop remarking on this, but it applies globally.